index.html

```diff
@@ -21,18 +22,149 @@
+                            <li>Back End Dev</li>
+                            <li>2015</li>
+                        </ul>
+                    </div>
+                    <p class="p-ul-details">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="ul-tags">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                <div class="work-btn"><button class="btn-in btn-anima">See Project</button></div>
+            </li>
+            <li class="project">
+                <img src="img/snap3.png" alt="Project Three">
+                <div class="text-work">
+                    <h2 class="work-title">Tonic</h2>
+                    <div class="work-details">
+                        <p class="p-details">CANOPY</p>
+                        <ul class="ul-details">
+                            <li>Back End Dev</li>
+                            <li>2015</li>
+                        </ul>
+                    </div>
+                    <p class="p-ul-details">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="ul-tags">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                <div class="work-btn"><button class="btn-in btn-anima">See Project</button></div>
+            </li>
+            <li class="project">
+                <img src="img/snap4.png" alt="Project Four">
+                <div class="text-work">
+                    <h2 class="work-title">Multi-Post Stories</h2>
+                    <div class="work-details">
+                        <p class="p-details">CANOPY</p>
+                        <ul class="ul-details">
+                            <li>Back End Dev</li>
+                            <li>2015</li>
+                        </ul>
+                    </div>
+                    <p class="p-ul-details">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="ul-tags">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                <div class="work-btn"><button class="btn-in btn-anima">See Project</button></div>
+            </li>
+        </ul>
+    </section>
+    <section class="about">
+        <div class="me">
+            <h2>About <br> Myself</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <a class="connect" href="#">LET'S CONNECT</a>
```

Comment:
- [ ] great job on the about section 👍 however the requirements mention that any title (e.g., "Let's connect") should use a header tag `<hx>`

Reply:
Done, now every title in the about section is in `<hx>`  tag.
I tried to find this requirement but I wasn't able to find it on the activity page, could you tell me where those requirements are?

Here is the link where I'm following the instructions:
https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md